src/installer/installer.go

```diff
@@ -106,7 +106,7 @@ func (i *installer) InstallNode() error {
 	// TODO report image to disk progress
 
 	err = utils.Retry(3, time.Second, i.log, func() error {
-		return i.ops.WriteImageToDisk(ignitionPath, i.Device, image)
+		return i.ops.WriteImageToDisk(ignitionPath, i.Device, image, i.ic)
```

Comment:
A conventions comment:
short names like 'i', 'ic' are OK for local scope
But on a large scope such as in the whole file, I think it is better to have a slightly better name